### DISCUSSION TITLE
Jandy Infinite WaterColors RS485 control?

### DISCUSSION THREAD

**Discussion by NWSAlpine** [tagyoureit/nodejs-poolController#913] [RS485]
The new Jandy Infinite WaterColors system looks like a more modern lighting system with direct color control and individually addressing lights. The transformer has RS485 control. Has anyone seen these yet to see if you can control it directly through poolcontroller or will these only work with AquaLink RS automation?

**Comment by tagyoureit** [comment]
We don't yet have support for it, but since it is controlled by RS485 it will be possible to add it.  We would need a willing participant to capture the RS-485 packets and record what functionality it represents and then it can be coded into the app.

But really, only 4 zones?  SMH.

**Reply by mguinness** [reply]
Related issue on https://github.com/aqualinkd/AqualinkD/issues/440.

**Reply by tagyoureit** [reply]
I just added some code for this based on what's available in the other thread.  Now need someone to test it.  Thanks for the link.
